packages/common/src/models/Analytics.ts

```diff
@@ -1647,6 +1648,11 @@ type ChatReportUser = {
   reportedUserId: ID
 }
 
+type ChatEntryPoint = {
+  eventName: Name.CHAT_ENTRY_POINT
+  source: 'banner' | 'navmenu' | 'share' | 'profile'
```

Comment:
no enum, wow!

Reply:
my main enum argument is for tracking usages of values, but this seems small enough to search for all `Name.CHAT_ENTRY_POINT` usages